controls/6_1_system_file_permissions.rb

```diff
@@ -83,8 +83,8 @@
   shadow_files.each do |f|
     describe file(f) do
       it { should exist }
-      it { should be_readable.by 'owner' }
-      it { should be_writable.by 'owner' }
+      it { should_not be_readable.by 'owner' }
```

Comment:
please add a comment describing our changes and the reason why they are divergent from the original

Reply:
please do this for any changes you make